tests/controllers/users.test.ts

```diff
@@ -50,17 +52,27 @@ describe('Test if Users CRUD operations are working correctly', () => {
     });
 
     test('Adding a new User should return the new User', async () => {
-        expect.assertions(7);
+        expect.assertions(6);
         const addedUser = await addUser(newUser);
         expect(typeof addedUser.id).toBe('number');
```

Comment:
Whilst it is best practice to only assert one thing per test I think for this specific case I agree with this pattern.

Reply:
When Typescript will be implemented we can check it with one assertion luckily